index.md

```diff
@@ -1,49 +1,41 @@
 
-## Where to start
+As well as modules, Origami provides the following services:
+
+- [Polyfill.io](http://polyfill.io) - Upgrade the web automatically, code using the latest standards when writing JavaScript applications
+- [The Image Service](http://image.webservices.ft.com/) - Deliver pictures in many formats and sizes; optimised and delivered over a a CDN.
+
+Origami services and modules all comply with the [Origami Spec](/spec) and [Coding Standards](/coding ) to ensure they are high quality and interoperable. All modules are developed in the open on [GitHub](http://github.com/Financial-Times).
+
+<h2> <i class="fa fa-wrench"></i> Origami tools</h2>
+
+To help you use the modules and services in your product there are the following tools:
+
+- [Origami Build Tools](https://github.com/Financial-Times/origami-build-tools) - Build Origami components as part of your existing build process
+- [Origami Build Service](dhaldas) - Deliver components directly to your user's browser using the build service
+- [The Registry](dsf) - Find and use the components you need from our component registry
+
+<h2> <i class="fa fa-book"></i> The Spec</h2>
+
+Origami is built according to the [Origami Spec](/docs/component-spec/). This ensures high quality and interoperability across modules. If you want to contribute to Origami modules or services, reading The Spec is a good place to begin.
 
-* If you're a non-technical stakeholder wondering what all the fuss is about and how Origami will affect you, read the [overview]({{site.baseurl}}/docs/overview/non-technical), which explains what problems we are solving and how Origami will help
-* If you're a product developer, read the [developer guide]({{site.baseurl}}/docs/developer-guide) to understand how to use Origami components in your product.
-* If you're a component developer, read the [component spec]({{site.baseurl}}/docs/component-spec) which sets out the rules that your components must conform to in order to be Origami components.
 
 ## Who is using Origami?
 
-Origami usage is widely spread across the business, as most of the teams at the FT are reporting to be using at least part of it:
-
-<div class="o-grid-row">
-	<div data-o-grid-colspan="6">
-		<ul>
-			<li><a href="http://www.ft.com">FT.com</a></li>
-			<li>Next FT</li>
-			<li><a href="http://app.ft.com">FT Web App</a></li>
-			<li><a href="http://ig.ft.com">Interactive Graphics</a></li>
-			<li>The membership team (barriers, registration…)</li>
-		</ul>
-	</div>
-	<div data-o-grid-colspan="6">
-		<ul>
-			<li>Specialist Titles (e.g. <a href="https://live.ft.com">live.ft.com</a>)</li>
-			<li>FT Salesforce developers</li>
-			<li>Business Metrics Intelligence (tracking and analytics)</li>
-			<li>Ads</li>
-			<li>Comments</li>
-		</ul>
-	</div>
-</div>
-
-Origami is also used by a range of third parties and even other entities such as Pearson Higher Education:
+Origami is used widely throughout the FT and by a range of third parties:
```

Comment:
I'm not sure anyone is actually interested in this information except the origami team, so saying summarised by "widely throughout the FT" seems enough. This way of expressing it means we don't need to maintain a list of people using it within the FT either.